### ISSUE TITLE
Enable Shortcuts On Page function interferes with comments

### ISSUE THREAD

**Issue by Synetech** [YePpHa/YouTubeCenter#1862]
You may want to mention in the [*Enable shortcuts on page*](https://github.com/YePpHa/YouTubeCenter/wiki/Features#enable-shortcuts-on-page) section of the manual that it will interfere with comments. For example, when trying to type a comment, every time you press space, it will start/stop playing the video. A note about that would clear up confusion of why that is happening.

**Comment by Owyn** [comment]
yes, the bug is still there, can't use spacebar in comment answers ='(
